### ISSUE TITLE
Feature/ Visualise the Aggregate history to as a changelog

### ISSUE THREAD

**Issue by rhyslbw** [meteor-space/todos#15]
This would be a great start to exposing the reasons behind event sourcing. The most simple form of this is just to have an additional view that shows each change to the Todo list in real-time, and we can then build on this over time to say rewind the app state to a point in the history. I understand there was the intention to make this app compliant for TodoMVC, but maybe we can divert that focus to https://github.com/meteor-space/TodoMVC and make this our playground and marketing tool.

It's pretty simple to achieve this, we have a pretty [basic package](https://github.com/meteor-space/event-sourcing-debug) already to publish out the commit collection, so we just need a polished view that makes the experience pleasant.

**Comment by rhyslbw** [comment]
I've updated the debug package and have created an [issue](https://github.com/meteor-space/event-sourcing-debug/issues/2) that relates to what's proposed here. Thoughts @darko-mijic ?

**Comment by rhyslbw** [comment]
BTW I'll release the new package version today

**Comment by darko-mijic** [comment]
I have reviewed your work and i agree that we should proceed in those lines with todos. I was planing to submit todos to TodoMVC as one of the compliant implementations but i don't think we need that kind of attention. Much more important is to make Space Todos a good showcase of Space ES infrastructure without complex domain and as a complete, non-distributed ES solution. 

I have tested one solution for better presentation of JSON data for ES debugging/demonstration. I will submit a PR to ES debug project.

**Comment by rhyslbw** [comment]
> Much more important is to make Space Todos showcase of Space ES infrastructure without complex domain and as a complete, non-distributed ES solution.

Exactly, todos apps are used for a reason for this ground level demo! We just need to expose the persistence strategy, and add features that demonstrate the reason for using ES, like projection rebuilding :sunglasses: 

> I have tested one solution for better presentation of JSON data for ES debugging/demonstration.

Great. I intentionally just did the minimum to bring it up to date, adding client-side structure to support more rapid development once we decide how to proceed with this being a more useful tool. Need to split the actual view layer out since this is currently opinionated for Blaze Components
